spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/config/DataFlowControllerAutoConfiguration.java

```diff
@@ -124,6 +125,7 @@ public MavenResourceLoader MavenResourceLoader(MavenProperties properties) {
 	}
 
 	@Bean
+	@ConditionalOnMissingBean
```

Comment:
I believe this should be: `@ConditionalOnMissingBean(DelegatingResourceLoader.class)`